src/Form/MembershipForm.php

```diff
@@ -19,6 +19,9 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     /* @var $entity \Drupal\membership\Entity\Membership */
     $form = parent::buildForm($form, $form_state);
 
+    // Hide revision message and revision fields...
+    $form['revision_information']['#access'] = FALSE;
+    $form['revision_log_message']['#access'] = FALSE;
```

Comment:
Hi,

There's certainly a use case for admins leaving comments, should probably change this to a permission. It's quite confusing with these fields visible, though, and for some reason it's showing up twice.

For now was just hiding to make this usable.